pkg/cmos/cmos.go

```diff
@@ -16,18 +16,34 @@ const (
 	cmosDataPort = 0x71
 )
 
+type CMOSChip struct {
```

Comment:
this is so close! Just one last question:
go folks would complain that names like
cmos.CMOSChip
cmos.cmosDataPort
cmos.cmosRegPort
"stutter"
I am going to leave this to you, I am approving this: could you consider changing a few names:
cmos.dataPort
cmos.regPort
cmos.Chip

Your call, best we get this in.

Reply:
Changes done. Squashed it so all related changes exist in one commit which can be reverted easily if required.